sites/hashdev/src/_pages/blog/0011_announcing-hook-service.mdx

```diff
@@ -0,0 +1,146 @@
+---
+authors:
+  - name: Ciaran Morinan
+    jobTitle: Head of Engineering, HASH
+    photo: blog/authors/ciaran-morinan.jpg
+  - name: Nate Higgins
+    jobTitle: Senior Platform Engineer, HASH
+    photo: blog/authors/nate-higgins.jpg
+postPhoto: blog/0004_announcing-error-stack/ErrorStack_Cover.jpg
+title: "Announcing the Block Protocol hook service"
+subtitle: Enabling applications to inject rich experiences inside blocks
+date: "2022-10-20"
+---
+
+## The world at your fingertips
+
+Open up an application and start typing.
+
+There’s a good chance that wherever you’re typing offers a world of possibilities beyond just plain text.
+
+Inputs and dropdowns are summoned by hotkeys, ready to inject links to people, pages, places.
+
+Formatting bars wait dutifully nearby, seen or unseen, ready to elevate your prose.
+
+The text and the space around it can brighten your screen with a rainbow of colours.
+
+## Behind the curtain
+
+We don’t often stop to think about how much magic underlies all this.
+
+A single formatted paragraph can be made up of dozens of pieces, each with a constellation of metadata.
+
+Any combination of fragments of those pieces can be selected, and transformations demanded.
+
+As we type, keystrokes are intercepted, inputs rendered, databases searched, relationships created.
+
+The average user is blissfully unaware of any of this complexity.
+
+They only notice when something they are familiar with is missing, or doesn’t work as expected.
+
+When the hotkeys painstakingly learned in one application do nothing, or — even worse — do something else. _Ugh_. Who decided _that_?
+
+A good text editing experience makes you forget it’s there.
+
+It makes you forget that the humble rectangle you are typing inside packs in more features, pixel for pixel, than anything else around it.
+
+## Textual deviance
+
+Text editing varies across applications: what features are available, and how the text is stored.
+
+Block-based applications are no different. Think WordPress, Contentful, Notion.
+
+At HASH, we’re helping develop a new open standard: the Block Protocol. Its goal is to allow any [block](https://blockprotocol.org/docs#what-is-a-block) to be used in any application, with zero configuration
+
+It does this by standardizing how blocks and applications communicate, and what messages they exchange.
+
+Because rich text editing is a core part of many modern web applications, we knew the Block Protocol had to support it.
+
+We had to decide how blocks made by different people could provide a consistent text editing experience _within_ an application, while allowing for variations in the experience **\***across**\*** applications – without the block knowing anything about the application it’s being used in.
```

Comment:
This is causing some weirdness

<img width="680" alt="Screenshot of rendered site being weird" src="https://user-images.githubusercontent.com/6226576/197261795-34ed3fa4-692b-4e87-adee-46aaf251aa40.png">

Reply:
https://github.com/hashintel/hash/pull/1241/commits/a1cab063dfff276d856ab957c2110263e5b9ea60